examples/miral-shell/miral-app.sh

```diff
@@ -77,8 +77,14 @@ unset QT_QPA_PLATFORMTHEME
 
 if [ "${miral_server}" == "miral-shell" ]
 then
+  if [ "$(lsb_release -c -s)" == "xenial" ]
+  then
+    export MIR_SERVER_APP_ENV=GDK_BACKEND=x11:QT_QPA_PLATFORM=ubuntumirclient:SDL_VIDEODRIVER=mir:NO_AT_BRIDGE=1
```

Comment:
To confirm, both GTK's Wayland *and* Mir backends are worse on Xenial than our XWayland support currently is?

Reply:
Confirmed.
1. Wayland doesn't work as GTK3 relies on xdg_shell_v5, which 
2. gtk-mir wasn't done in Xenial (was better in 17.04) and seems to have suffered bitrot since.